docs/troubleshooting.md

```diff
@@ -0,0 +1,130 @@
+
+```
+$ oc get pods -n openshift-marketplace
+
+NAME                                    READY     STATUS    RESTARTS   AGE
+certified-operators-79f876c76d-mbtjg    1/1       Running   0          86m
+community-operators-854c564f5f-9vk2x    1/1       Running   0          86m
+marketplace-operator-6dcd66cb8c-2wmkz   1/1       Running   0          86m
+redhat-operators-647fd668fc-9ff5d       1/1       Running   0          86m
+
+```
+
+If any pod is in a state other than `Running` inspect the logs of the pod with `oc logs <Name-of-Pod>`.
+
+If all pods look healthy, make sure that there's a CatalogSource for each corresponding CatalogSourceConfigs.
+
+```
+$ oc get catalogsource -n openshift-marketplace 
+
+NAME                  NAME                  TYPE      PUBLISHER   AGE
+certified-operators   Certified Operators   grpc      Red Hat     23h
+community-operators   Community Operators   grpc      Red Hat     23h
+redhat-operators      Red Hat Operators     grpc      Red Hat     23h
+
+```
+If you need help debugging the CatalogSources, please reach out to the [#forum-operator-fw](https://coreos.slack.com/messages/C3VS0LV41/) slack channel.
+
+Next, make sure that the `packagemanifests` were created by the `package-server` pods. 
+
+```
+$ oc get packagemanifests -n openshift-marketplace
+
+NAME                             CATALOG               AGE
+packageserver                    OLM Operators         3h31m
+
+```
+
+If no `packagemanifests` were created, check the logs for the `package-server` pods.
+
+```
+$ oc get pods -n openshift-operator-lifecycle-manager
+
+NAME                                READY     STATUS    RESTARTS   AGE
+catalog-operator-5c8cd86959-vgbmm   1/1       Running   0          3h57m
+olm-operator-7b945f5dcf-xxnsk       1/1       Running   0          3h57m
+olm-operators-cnxw9                 1/1       Running   0          3h54m
+packageserver-6c664f6d76-jj24b      1/1       Running   0          3h54m
+packageserver-6c664f6d76-kzpd9      1/1       Running   0          3h53m
+
+$ oc logs packageserver-6c664f6d76-jj24b -n openshift-operator-lifecycle-manager
+$ oc logs packageserver-6c664f6d76-kzpd9 -n openshift-operator-lifecycle-manager
+
+```
+
+The logs for the `package-server` pods should contain information about why it might have failed to detect the presence of the `CatalogSources`. If the `package-server` pod does not show up or is crashing (anything other than healthy), try killing the pod or editing the deployment you see when you `oc get deployments -n openshift-operator-lifecycle-manager`.
+
+If everything seems healthy, and still no packages show up in the UI, it could be a browser issue. Check the the browser console to see the logs for possible errors. At any point in the steps above if the error looks too complicated to debug, please reach out to the [#forum-ui slack channel](https://coreos.slack.com/messages/C6A3NV5J9). 
```

Comment:
Isn't coreos a Red Hat internal slack?

Reply:
@awgreene I suppose whenever someone click's on the link it leads to a login page? Is there anywhere else non Red Hat users can reach out to for help at the moment besides opening an issue?

Reply:
@anik120 You can't get into that slack instance without a redhat login. The only publicly available places that I am aware of are the #forum-operators channel in the upstream Kubernetes slack instance (https://slack.k8s.io/) and there is also a google group that is specifically for the Operator-Framework (https://groups.google.com/forum/#!forum/operator-framework). BTW if you aren't already subscribed to both of those I would suggest you do so.

Reply:
There is a `kubernetes-operators` channel in the kubernetes slack and an operator email list. Perhaps @kevinrizza @galletti94 have opinions on what should be included here?

Reply:
> There is a `kubernetes-operators` channel in the kubernetes slack and an operator email list. Perhaps @kevinrizza @galletti94 have opinions on what should be included here?

See my above reply. I think referencing both of those is appropriate here.

Reply:
@awgreene @kevinrizza changed links of slack channels to Kubernetes slack and Operator-framework google group.